src/createServer.js

```diff
@@ -1,11 +1,25 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+
+const usersService = require('./services/users');
+const expensesService = require('./services/expenses');
+
+const usersRouter = require('./routes/users');
+const expensesRouter = require('./routes/expenses');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  usersService.getInitial();
+  expensesService.getInitial();
+
+  const app = express();
+
+  app.use(cors());
+  app.use('/users', express.json(), usersRouter);
+  app.use('/expenses', express.json(), expensesRouter);
```

Comment:
Can be moved to separate `use`

```suggestion

  app.use(cors());
  app.use(express.json());
  app.use('/users', usersRouter);
  app.use('/expenses', expensesRouter);
```